www/__tests__/enketoHelper.test.ts

```diff
@@ -0,0 +1,245 @@
+  const xmlDoc2 = xmlParser.parseFromString(xmlString2, 'text/xml');
+
+  //if no template, returns "Answered" TODO: find a way to engineer this case
+  //if no labelVars, returns template TODO: find a way to engineer this case
+  //have a custom survey label function TODO: we currently don't have custome label functions, but should test when we do
+  //no custom function, fallback to UseLabelTemplate (standard case)
+  expect(await resolveLabel('TimeUseSurvey', xmlDoc)).toBe('3 Domestic');
+  expect(await resolveLabel('TimeUseSurvey', xmlDoc2)).toBe('3 Employment/Education, 3 Domestic');
+});
+
+/**
+ * @param surveyName the name of the survey (e.g. "TimeUseSurvey")
+ * @param enketoForm the Form object from enketo-core that contains this survey
+ * @param appConfig the dynamic config file for the app
+ * @param opts object with SurveyOptions like 'timelineEntry' or 'dataKey'
+ * @returns Promise of the saved result, or an Error if there was a problem
+ */
+//   export function saveResponse(surveyName: string, enketoForm: Form, appConfig, opts: SurveyOptions) {
+it('gets the saved result or throws an error', () => {
+  const surveyName = 'TimeUseSurvey';
+  const form = {
+    getDataStr: () => {
+      return '<aDxjD5f5KAghquhAvsormy xmlns:jr="http://openrosa.org/javarosa" xmlns:orx="http://openrosa.org/xforms" id="time_use_survey_form_v9_1"><start>2023-10-13T15:05:48.890-06:00</start><end>2023-10-13T15:05:48.892-06:00</end><group_hg4zz25><Start_date>2016-07-25</Start_date><Start_time>17:24:32.928-06:00</Start_time><End_date>2016-07-25</End_date><End_time>17:30:31.000-06:00</End_time><Activity_Type>personal_care_activities</Activity_Type><Personal_Care_activities>doing_sport</Personal_Care_activities><Employment_related_a_Education_activities/><Domestic_activities/><Recreation_and_leisure/><Voluntary_work_and_care_activities/><Other/></group_hg4zz25><meta><instanceID>uuid:dc16c287-08b2-4435-95aa-e4d7838b4225</instanceID><deprecatedID/></meta></aDxjD5f5KAghquhAvsormy>';
+    },
+  };
+  const badForm = {
+    getDataStr: () => {
+      return '<aDxjD5f5KAghquhAvsormy xmlns:jr="http://openrosa.org/javarosa" xmlns:orx="http://openrosa.org/xforms" id="time_use_survey_form_v9_1"><start>2023-10-13T15:05:48.890-06:00</start><end>2023-10-13T15:05:48.892-06:00</end><group_hg4zz25><Start_date>2016-08-25</Start_date><Start_time>17:24:32.928-06:00</Start_time><End_date>2016-07-25</End_date><End_time>17:30:31.000-06:00</End_time><Activity_Type>personal_care_activities</Activity_Type><Personal_Care_activities>doing_sport</Personal_Care_activities><Employment_related_a_Education_activities/><Domestic_activities/><Recreation_and_leisure/><Voluntary_work_and_care_activities/><Other/></group_hg4zz25><meta><instanceID>uuid:dc16c287-08b2-4435-95aa-e4d7838b4225</instanceID><deprecatedID/></meta></aDxjD5f5KAghquhAvsormy>';
```

Comment:
Add comment explaining why this form is bad

Reply:
done in [7ec575e](https://github.com/e-mission/e-mission-phone/pull/1063/commits/7ec575e41701725723609c9eb40654ef8d05385b)